README.rst

```diff
@@ -235,6 +236,44 @@ Delete
     print(response)
     # <class pybutton.Response >
 
+Customers
+~~~~~~
+
+Create
+''''''
+
+.. code:: python
+
+    import hashlib
+    from pybutton import Client
+
+    client = Client('sk-XXX')
+
+    hashed_email = hashlib.sha256('user@example.com'.lower().strip()).hexdigest()
+
+    response = client.customers.create({
```

Comment:
ah, my mistake.  I see how `email(s)_sha256` is picked up, but still not `advertising_id.